### DISCUSSION TITLE
StartBootstrap Agency CSS Free OpenSource Theme Support

### DISCUSSION THREAD

**Discussion by johnnyq** [givanz/Vvveb#178] [General]
Hello I currently use the Free and Opensource Agency Bootstrap 5 CSS Theme for a few of my static sites here https://github.com/StartBootstrap/startbootstrap-agency
What is the best way to import the theme into Vvveb?
Thank you, just found out about this opensource project today and so far loving it.

**Comment by givanz** [comment]
Hi

Thank you.

You can use upload theme function and upload a zip with the [dist folder](https://github.com/StartBootstrap/startbootstrap-agency/tree/master/dist).

![theme-upload](https://github.com/user-attachments/assets/28de69b7-f6c6-4e78-8a9a-fa1ec3a80287)

Or you can copy the `/dist` folder to `/public/themes` 

Note, if you use multi site and use the same theme for more than one site then theme/layout changes made from one site will be visible on all sites.

To avoid this use duplicate theme button to create a new theme for a site.

**Comment by johnnyq** [comment]
Wow thank you so much for your fast and detailed response @givanz I will give this a shot.
